<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAIPFPropertiesListDialog</class>
 <widget class="QDialog" name="CAIPFPropertiesListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Pathfinding Properties List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,4,0,1,0">
   <item>
    <widget class="QLabel" name="treeLabel">
     <property name="text">
      <string>Choose AI Pathfinding Properties:</string>
     </property>
     <property name="buddy">
      <cstring>listView</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="listView">
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Edit...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>descriptionEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descriptionEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="selectionListEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>selectionListEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CAIPFPropertiesListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CAIPFPropertiesListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
